jsonschema-core.xml

```diff
@@ -3846,8 +3859,8 @@ https://example.com/schemas/common#/$defs/count/minimum
                             <t>"$schema" MAY change for embedded resources</t>
                             <t>Array-value "items" functionality is now "prefixItems"</t>
                             <t>"items" subsumes the old function of "additionalItems"</t>
-                            <t>"contains" and "unevaluatedItems" interactions now specified</t>
-                            <t>Rename $recursive* to $dynamic*</t>
+                            <t>"contains" annotation behavior, and "contains" and "unevaluatedItems" interactions now specified</t>
```

Comment:
I'm not quite sure what you're saying here. Isn't correcting errata the entire point of this interim revision?

Reply:
Change logs, like the IETF documents they describe, are not mutable.  This is the change log for `draft-bhutton-json-schema-00`.  We are now working on `draft-bhutton-json-schema-01`, so any changes have to go in the change log section for that.  If you think the old change log was inadequate because it didn't explicitly mention "annotation behavior", well, change logs are not precise.  They're just a few bullet points to call attention to the main changes.

Usually after every major draft I went to a great deal of effort to write the full migration guide on the web page.  Which I assume is why there's still no migration guide for 2020-12 since I guess no one else has had time to write one.   That's where the details go.

Reply:
I think this change is perfectly fine. Change logs can have errata too and I don't see a problem with fixing those issues in a patch release. I don't think this is a critical change, but it would be nice to have.

FYI, @handrews, I'm working on the migration guide for 2020-12 right now :slightly_smiling_face:

Reply:
I apologise profusely for not having been able to publish the spec AND have a migration guide out within a reasonable time.

Reply:
It's a pretty small clarification to the chagelog, so I think that's fine.